docker-entrypoint.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+set -eu
+
+# This makes environment variables available to everyone, including crond and its crons
+env > /etc/environment
+
+# Configuration
+CRON=${CRON:-}
+CRON_UID=${CRON_UID:-0} # Default is 'root' user
+CRON_GID=${CRON_GID:-0} # Default is 'root' user
+CRON_USER=$( (getent passwd "$CRON_UID" || echo 'cronuser') | cut -d ':' -f1 )
+if [ "$CRON_UID" = "$CRON_GID" ]; then
+    CRON_GROUP="$CRON_USER"
+else
+    CRON_GROUP=$( (getent group "$CRON_GID" || echo 'crongroup') | cut -d ':' -f1 )
+fi
+echo "Crons will run under user '$CRON_USER' of UID '$CRON_UID' and group '$CRON_GROUP' of GID '$CRON_GID'"
+
+# Validation
+if [ -z "$CRON" ]; then
+    echo "CRON is empty."
+    exit 1
+fi
+
+# Create user, group, and add membership
```

Comment:
I think user/group should be created and then set on the Dockerfile level. This will allow this script to be ran as a normal user rather than root. wdyt?

Reply:
Fixed